kubernetes/annotations_and_labels.md

```diff
@@ -41,19 +41,30 @@ This page defines common annotations and labels we set in Kubernetes objects.
   managed by `chart-operator`.
 - `release.giantswarm.io/version` - value should be Giant Swarm release
   version, e.g. `release.giantswarm.io/version=2.3.0`.
-- `OPERATOR.giantswarm.io/version` - value should be an operator version, e.g.
-  `kvm-operator.giantswarm.io/version=1.0.0`. This is not the same as release
-  version. This value can be the same in multiple releases. This label is used
-  by the operator to recognize which object it should reconcile. When set on
-  nodes it is used to set that information in the status with the
-  statusresource.
+- `app.giantswarm.io/version` - value should be an app/operator version as
+  defined in `project.go`, e.g. `app.giantswarm.io/version=1.0.0`. This is not
+  the same as release version. This value can be the same in multiple releases.
+  This label is used by the operator to recognize which object it should
+  reconcile. When set on nodes it is used to set that information in the status
+  with the statusresource.
+- `app.giantswarm.io/branch` - (informational) branch from which this
+  instance of the app/operator was built from.
+- `app.giantswarm.io/commit` - (informational) ID of the commit from which
+  this instance of the app/operator was built from.
 - `giantswarm.io/provider` - value should be the installation's provider, e.g.
   `kvm`, `aws`, or `azure`.
 - `version` - value should contain the version of the application.  Should be applied
   to every Kubernetes resource associated with an application. This together with `app`
   label should also be used as replicas selector. E.g. `app=kvm-operator,version=1.0.0`. Exceptions can
   be made to accomodate for adherence to existing selectors upstream.
```

Comment:
With `app.giantswarm.io/version` we should update it and state that this is the chart version.

```suggestion
- `version` - value should contain the version of the chart installing the
  application.  Should be applied to every Kubernetes resource in the chart.
  This together with `app` label should also be used as replicas selector. E.g.
  `app=kvm-operator,version=1.0.0`. Exceptions can be made to accommodate for
  adherence to existing selectors upstream.
```

Reply:
Good catch :+1: